Samples/JavaKitSampleApp/Sources/JavaKitExample/JavaKitExample.swift

```diff
@@ -77,7 +77,12 @@ extension HelloSwift: HelloSwiftNativeMethods {
     }
 
     // Make sure that the thread safe class is sendable
-    let threadSafe: Sendable = ThreadSafeHelperClass(environment: javaEnvironment)
+    let helper = ThreadSafeHelperClass(environment: javaEnvironment)
+    let threadSafe: Sendable = helper
+
+    let text: String? = helper.textOptional
```

Comment:
since we start having a lot of things here maybe put this into `void testOptionals()` and have a bunch of checks there, including checking the numeric ones?